components/radioButton/radioButton.scss

```diff
@@ -0,0 +1,61 @@
+.ui-radio {
+  color: $tx-generic-color-text-dark;
+  font-family: $tx-generic-font-secondary-font-family, $tx-generic-font-secondary-generic-family;
+  font-size: 14px;
+  font-weight: $tx-generic-font-primary-weight-regular;
+  min-height: 20px;
+  padding-bottom: 2px;
+  padding-right: 10px;
+  padding-top: 2px;
+  position: relative;
+
+  & input[type="radio"] {
+    left: -9999px;
+    position: absolute;
+
+    & + label {
```

Comment:
@ibsukru how do you suggest to use the pseudo selectors using the  mentioned class? I am bit concerned because  i need to combine multiple elements with their pseudo selectors, for ex. `input[type="radio"]:checked + label:after`. In this case I will have to write something like `input[type="radio"]:checked + label.ui-radio__label:after`